src/module-1/cards-list-v1/styles.css

```diff
@@ -0,0 +1,93 @@
+* {
```

Comment:
You need to fix css styles to display a list of products at the center of the screen. Currently there is too much space between each product.